project-security/verifying-signatures.md

```diff
@@ -480,22 +480,18 @@ malicious ISO, computed the hash values for that malicious ISO, and replaced
 the values in `Qubes-RX-x86_64.iso.DIGESTS` with his own set of values.
 Therefore, we should also verify the authenticity of the listed hash values.
 Since `Qubes-RX-x86_64.iso.DIGESTS` is a clearsigned PGP file, we can use GPG
-to verify it from the command line:
-
-1. [Import and authenticate the Qubes Master Signing Key.](#how-to-import-and-authenticate-the-qubes-master-signing-key)
-2. [Import and authenticate your release signing key.](#how-to-import-and-authenticate-release-signing-keys)
-3. Verify the signature in the digest file:
-
-   ```shell_session
-   $ gpg2 -v --verify Qubes-RX-x86_64.iso.DIGESTS
-   gpg: armor header: Hash: SHA256
-   gpg: armor header: Version: GnuPG v2
-   gpg: original file name=''
-   gpg: Signature made Tue 20 Sep 2016 10:37:03 AM PDT using RSA key ID 03FA5082
-   gpg: using PGP trust model
-   gpg: Good signature from "Qubes OS Release X Signing Key"
-   gpg: textmode signature, digest algorithm SHA256
-   ```
+to verify the signature in the digest file:
+
+```shell_session
+$ gpg2 -v --verify Qubes-RX-x86_64.iso.DIGESTS
+gpg: armor header: Hash: SHA256
+gpg: armor header: Version: GnuPG v2
+gpg: original file name=''
+gpg: Signature made Tue 20 Sep 2016 10:37:03 AM PDT using RSA key ID 03FA5082
+gpg: using PGP trust model
+gpg: Good signature from "Qubes OS Release X Signing Key"
+gpg: textmode signature, digest algorithm SHA256
+```
```

Comment:
Why not repeat those steps here? Some people will surely miss them and ask silly questions as a result. In fact, this has already happened several times.

Reply:
Those steps are listed as prereqs in the narrowest linkable scope, so if they're not going to read those, then they're probably equally likely not to read these.  Also, the flow of repeating those steps here reads awkwardly - we're introduced with "we can use GPG to verify it from the command line:", and then there's an unspoken "wait, before you do that, did you remember to do these prereqs?"

Reply:
I do think some people will miss it or forget without the reminder, but perhaps it's best not to let such readers' shortcomings unduly influence the writing. Let's try it your way. 🙂